spring-cloud-gcp-samples/checkstyle-samples-suppressions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+    "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<suppressions>
+  <suppress files=".*" checks="LineLength" />
+  <suppress files=".*\.java" checks="HideUtilityClassConstructorCheck" />
+
+  <!-- Sample apps should be usable by someone without correct import order IDE settings. -->
+  <suppress files=".*\.java" checks="ImportOrder"/>
```

Comment:
I think the committers should still be bound by strict style checks, even in the samples.
If the concern is for folks checking out the code and playing with the samples, and getting tripped by checkstyle, we should move checkstyle to a Maven profile, and only enable it on Travis. This will ensure that casual users of the samples are not tripped by it, but the style is still enforced on commits.